tests/baselines/reference/catchClauseWithUnknownTypeAnnotation.types

```diff
@@ -0,0 +1,6 @@
+=== tests/cases/compiler/catchClauseWithUnknownTypeAnnotation.ts ===
+try {
+} catch (e: unknown) {
+>e : any

```

Comment:
Seems like the type annotation is not working as expected.

Reply:
Ohh, overlooked. I'll check.

Reply:
@ajafff I've fixed.